_maps/map_files/MetaStation/MetaStation.dmm

```diff
@@ -74008,7 +74025,6 @@
 	},
 /area/maintenance/aft2)
 "nkU" = (
-/mob/living/simple_animal/pet/dog/corgi/borgi,
```

Comment:
You have unrelated inter-key changes on meta still, we do not want more key content shuffling.